$NetBSD: patch-ao,v 1.1 2007/01/05 13:20:54 obache Exp $

Prevent amverify from looping forever. Patch provided by Jean-Louis
Martineau on amanda-users:
http://groups.yahoo.com/group/amanda-users/message/55099

--- server-src/amverify.sh.in.orig	2004-10-04 11:47:06.000000000 +0000
+++ server-src/amverify.sh.in
@@ -428,6 +428,7 @@ while [ $SLOT -lt $SLOTS ]; do
 		FILE=`grep restoring $TEMP/amrestore.out \
 			| sed 's/^.*restoring //'`
 		EOF=`grep "reached end of tape" $TEMP/amrestore.out`
+		EOI=`grep "reached end of information" $TEMP/amrestore.out`
 		# amrestore:   0: restoring sundae._mnt_sol1_usr.19961127.1
 		if [ X"$FILE" != X"" -a X"$RESULT" = X"0" ]; then
 			report "Checked $FILE"
@@ -436,6 +437,9 @@ while [ $SLOT -lt $SLOTS ]; do
 		elif [ -n "$EOF" ]; then
 			report "End-of-Tape detected."
 			break
+		elif [ -n "$EOI" ]; then
+			report "End-of-Information detected."
+			break
 		else
 			report "** Error detected ($FILE)"
 			echo "$VOLUME ($FILE):" >>$DEFECTS
